source/errors.js

```diff
@@ -59,7 +59,10 @@ module.exports.ParseError = class extends GotError {
 };
 
 module.exports.HTTPError = class extends GotError {
-	constructor(statusCode, statusMessage, headers, opts) {
+	constructor(request, opts) {
```

Comment:
Shouldn't parameter 1 be `response` instead of `request`?

Reply:
changed